.github/scripts/generate_ci_workflows.py

```diff
@@ -504,6 +504,16 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
             labels=set([LABEL_CIFLOW_DEFAULT, LABEL_CIFLOW_LINUX, LABEL_CIFLOW_CUDA]),
         ),
     ),
+    CIWorkflow(
+        arch="linux",
+        build_environment="linux-xenial-cuda11.3-py3.6-gcc7-no-ops",
+        docker_image_base=f"{DOCKER_REGISTRY}/pytorch/pytorch-linux-xenial-cuda11.3-cudnn8-py3-gcc7",
+        test_runner_type=LINUX_CUDA_TEST_RUNNER,
+        exclude_test=True,
+        ciflow_config=CIFlowConfig(
+            labels=set([LABEL_CIFLOW_DEFAULT, LABEL_CIFLOW_LINUX, LABEL_CIFLOW_CUDA]),
```

Comment:
In general I'd like to limit the number of CUDA workflows that we run by default in PR workloads so I'd suggest removing the LABEL_CIFLOW_DEFAULT from here.

This means that this workflow would only run on master but could be enabled using the `ciflow/cuda` label.

Reply:
```suggestion
            labels=set([LABEL_CIFLOW_LINUX, LABEL_CIFLOW_CUDA]),
```

Reply:
>This means that this workflow would only run on master but could be enabled using the ciflow/cuda label.

Running on PRs is the whole point. By the time code lands in master it's already passed fb-internal testing.

I've added a cpu-only build to run in default and a separate cuda one for `ciflow/cuda`. Hopefully that's okay.

Reply:
@seemethere @malfet can you  clarify what the issue is with cuda CI jobs:
- If it's CUDA hardware, then since this doesn't run any tests I think it should be able to run without a GPU.
- If it's because `nvcc` is slow, then setting `TORCH_CUDA_ARCH_LIST` might help to speed up `nvcc`.
- Or, if that's still too much maybe it's possible to have the default build only run the pre-processor and not fully build. That would be a lot faster but still catch include errors.

Thoughts?